nuxt/framework
# refactor!: rename nuxt3 to nuxt

### 🔗 Linked issue



### ❓ Type of change




### 📚 Description

Finally renaming `nuxt3` to `nuxt` in all internals in order to release as `nuxt@3.0.0-rc.1` (`nuxt@rc`)

Breaking change for edge users:

1. Update `package.json`

```diff
{
  "devDependencies": {
-    "nuxt3": "latest"
+    "nuxt": "npm:nuxt3@latest"
  }
}
```

2. Update `nuxt.config`:

```diff
- import { defineNuxtConfig } from 'nuxt3'
+ import { defineNuxtConfig } from 'nuxt'

export default defineNuxtConfig({ ... })
```

Todo:
- [x] Update internal references
- [x] Update examples
- [x] Detect 3.x via `nuxt` for kit utils 
- [x] Passing tests
- [x] Ensure release script is nice :}

### 📝 Checklist
